sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala

```diff
@@ -634,6 +634,39 @@ class DateTimeUtilsSuite extends SparkFunSuite with Matchers with SQLHelper {
     }
   }
 
+  test("SPARK-35099: microseconds to LocalDateTime") {
```

Comment:
JIRA ID is incorrect, please, change it.

Reply:
Done, thanks.